packages/cli/src/UserManagement/PermissionChecker.ts

```diff
@@ -112,6 +104,16 @@ export class PermissionChecker {
 				: `${subworkflowOwner.firstName} (${subworkflowOwner.email}) can make this change. You may need to tell them the ID of this workflow, which is ${subworkflow.id}`,
 		);
 
+		let policy =
+			subworkflow.settings?.callerPolicy ?? config.getEnv('workflows.callerPolicyDefaultOption');
+
+		if (!isSharingEnabled()) {
+			// Community version allows only same owner workflows
+			policy = 'workflowsFromSameOwner';
+			errorToThrow.description =
+				'In the community version, a subworkflow can only be called by a workflow created by the same owner as the subworkflow.';
```

Comment:
I'd refrain from using "Community" as the same message is displayed in cloud; I'd just say that the sharing is disabled and therefore subworkflows owned by other people cannot be called.
```suggestion
				'When sharing is disabled, only workflows owned by the same user can be called.';
```